text/paste.rs

```diff
@@ -8,53 +8,85 @@
 //
 // TODO:
 // - stdin ("-")
-// -- Probably fixed
+// -- Fixed
 // - fix:  empty-string delimiters \0
-// -- Probably fixed
+// -- Fixed
 // - improve:  don't open all files at once in --serial mode
 //
 
 use clap::Parser;
 use gettextrs::{bind_textdomain_codeset, setlocale, textdomain, LocaleCategory};
 use plib::PROJECT_NAME;
-use std::cell::Cell;
-use std::collections::hash_map::Entry;
-use std::collections::HashMap;
+use std::cell::{OnceCell, RefCell};
 use std::error::Error;
 use std::fs::File;
-use std::io::{self, BufRead, BufReader, Write};
+use std::io::{self, BufRead, BufReader, Stdin, Write};
+use std::iter::Cycle;
 use std::rc::Rc;
+use std::slice::Iter;
 
 /// paste - merge corresponding or subsequent lines of files
 #[derive(Parser)]
 #[command(version, about)]
 struct Args {
-    /// Concatenate all of the lines from each input file into one line of output per file, in command line order.
+    /// Concatenate all of the lines from each input file into one line of output per file, in command line order
     #[arg(short, long)]
     serial: bool,
 
     /// Delimiter list
-    #[arg(short, long)]
+    // Other implementations use "delimiters"
+    #[arg(alias = "delimiters", short, long)]
     delims: Option<String>,
```

Comment:
In terms of long options,
* POSIX does not specify, so posixutils has complete freedom of choice
* Default is to be compatible with in-the-field long options (GNU utils, typically)
* "delims" is our own invention presumably, and by the above-stated logic, should be changed to `delimiters`
* conclusion:  long="delimiters" seems like a better choice, WDYT?